Nautilus/Documentation/tutorials/csproj-tutorials.md

```diff
@@ -105,17 +105,22 @@ Still in the csproj, add the following outside of any PropertyGroups:
 
 You shouldn't have to change this.
 
-### Step 4
+#### Step 4
 
 Finally, once again in the csproj outside of any PropertyGroups, add the following:
 
 ```xml
 <!--Post-Build event that automatically places your mods folder with the DLL and documentation into your plugins folder as defined in GameDir.targets-->
 <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-	<Exec Command="copy /Y &quot;$(TargetPath)&quot; &quot;$(PluginsDir)\$(TargetName)&quot;" />
+	<Copy SourceFiles="$(TargetPath)" DestinationFolder="$(PluginsDir)\$(TargetName)" />
 </Target>
 ```
 
+If you want to do the same for a .pdb file, add this to just before `</Target>`:
+
+```xml
+    <Copy SourceFiles="$(TargetDir)\(TargetName).pdb" DestinationFolder="$(PluginsDir)\$(TargetName)" />
+```
 Make sure a folder exists in the BepInEx/plugins folder with the same name as your mod DLL.
```

Comment:
This is unnecessary. Simply add the following line before the `Copy` command and then a folder will be automatically made.
```xml
<MakeDir Directories="$(PluginsDir)\$(TargetName)"/>
```

Reply:
Short for "looks good to me"